src/components/atoms/SpacesDropdown/SpacesDropdown.tsx

```diff
@@ -19,12 +19,14 @@ const spaceNoneOption = Object.freeze({
   id: "",
   name: "",
   template: undefined,
+  host: { icon: "" },
```

Comment:
[Q] Why is `host` even used? Can it not be flat `icon` at the top level? Like

```tsx
const spaceNoneOption: SpacesDropdownPortal = {};
Object.freeze(spaceNoneOption);
```
Side note: name here should probably be like a constant, e.g. `SPACE_OPTION_NONE`

If  `host` must be there, we may need to add some Interface that is a kind of partial of the space.
In that case, please add a debt comment to formally define new interface and/or do an interface segregation of the target one

Reply:
using host this way matches the `space` entity type and doesn't trigger TS errors